push/src/main/java/org/aerogear/mobile/push/UnifiedPushConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.aerogear.mobile.push;
+
+import java.util.List;
+
+public final class UnifiedPushConfig {
+
+    private String alias;
+    private List<String> categories;
+
+    public String getAlias() {
+        return alias;
+    }
+
+    public void setAlias(String alias) {
+        this.alias = alias;
+    }
+
+    public List<String> getCategories() {
```

Comment:
Should we do a null check here and return an empty list if there are no categories?

Reply:
Done!